src/mame/drivers/apple2gs.cpp

```diff
@@ -4593,6 +4594,7 @@ static void apple2_cards(device_slot_interface &device)
 	device.option_add("uthernet", A2BUS_UTHERNET);  /* A2RetroSystems Uthernet card */
 	device.option_add("sider2", A2BUS_SIDER2); /* Advanced Tech Systems / First Class Peripherals Sider 2 SASI card */
 	device.option_add("sider1", A2BUS_SIDER1); /* Advanced Tech Systems / First Class Peripherals Sider 1 SASI card */
+	device.option_add("grapplerplus", A2BUS_GRAPPLERPLUS); /* Orange Micro Grappler Plus Parallel Interface card */
```

Comment:
Should this also be added for the IIgs and possibly other Apple 2 families?  There are lists of slot options hiding all over the place for Apple II cards.

Reply:
I think I added it to the apple2, 2e and 2gs drivers at the end of the option_add list.